src/Donations/LeftPanel/LeftPanel.module.scss

```diff
@@ -124,16 +124,16 @@
 }
 
 .project-title {
-  display: flex;
-  align-items: stretch;
-  flex-wrap: wrap;
-  column-gap: 10px;
+  a {
+    display: inline;
+  }
 }
 
 .verified-badge {
   display: flex;
   align-items: center;
   height: 1.43em; //hack to match line height of h1 tag
+  margin-bottom: -4px;
```

Comment:
Can this be avoided? I'm not in favour of using specific margin values to align an svg icon with text because adjustments need to be made for the icon if the text size changes.

This may be useful: https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1787/commits/40cd858eba669e966310c09f84985967b2ac3d0d#diff-c93408bc8daf261e6213c48e6213df65520309cc678c62393f2a58d6c2c08364